BUILD.gn

```diff
@@ -372,67 +375,79 @@ action("create_symbols_dist") {
 
 action("create_dist_zips") {
   output = "$brave_dist_dir/$brave_product_name-v$brave_version-$brave_platform-$target_arch.zip"
+  outputs = [ output ]
 
-  script = "//brave/script/create-dist.py"
+  if (is_win) {
+    # Repack a chrome release archive to the brave format.
+    # TODO(atuchin): support other platforms.
 
-  inputs = get_target_outputs(":brave_license_dist_resources")
+    script = "//brave/script/repack-archive.py"
+    deps = [ "//chrome/installer/mini_installer:mini_installer_archive" ]
 
-  if (!is_mac && !is_android && !is_ios) {
-    inputs += get_target_outputs(":brave_dist_resources")
-    inputs += get_target_outputs(":brave_locale_dist_resources")
-    if (enable_extensions) {
-      foreach(locale, locales) {
-        inputs += get_target_outputs(":brave_shields_locales_${locale}")
-        inputs += get_target_outputs(":brave_rewards_locales_${locale}")
+    input = "$root_out_dir/chrome.7z"
+    inputs = [ input ]
+
+    args = [
+      "--input=" + rebase_path(input),
+      "--output=" + rebase_path(output),
+      "--build_dir=" + rebase_path(root_out_dir),
+      "--target_dir=Chrome-bin",
```

Comment:
If this is static string, how about making it as a local var of repack-archive.py instead of passing?

Reply:
Ah! NVM. This dir depends on input file. Got it.
Different input file can have different target dir.

Reply:
Moreover, if we want to support Mac or Linux it become non-static.